QEfficient/exporter/export_utils.py

```diff
@@ -62,10 +63,14 @@ def export_onnx(
             dynamic_axes[iname] = {0: "batch_size", 1: "decoder_seq_len"}
         elif iname.startswith("past_"):
             # KV-cache (batch_size, num_heads, past_len, embed_dim)
-            dynamic_axes[iname] = {0: "batch_size", 2: "ctx_len"}
+            dynamic_axes[iname] = {0: dynamic_axis_past_key, 2: "ctx_len"}
+        elif iname == "batch_index":
+            dynamic_axes[iname] = {0: "batch_size"}
+
     if "past_key.0" in input_names and "attention_mask" in input_names:
         dynamic_axes["attention_mask"] = {0: "batch_size", 1: "ctx_len"}
 
+    custom_opsets = {"com.qti.aisw.onnx": 1} if dynamic_axis_past_key == "full_batch_size" else None
```

Comment:
Is this required?

Reply:
This can be removed. Will remove it from the the PR